src/openrct2/world/Sprite.cpp

```diff
@@ -689,26 +689,26 @@ void sprite_set_coordinates(int16_t x, int16_t y, int16_t z, rct_sprite* sprite)
  *
  *  rct2: 0x0069EDB6
  */
-void sprite_remove(rct_sprite* sprite)
+void sprite_remove(SpriteBase* sprite)
 {
-    auto peep = sprite->AsPeep();
+    auto peep = ((rct_sprite*)sprite)->AsPeep();
```

Comment:
This one sounds weird

Reply:
Yeah I really want to refactor so that `rct_sprite` is basically never used except when you need to know the max size of a sprite. For now this is just a stupid way of getting to the function I want.